### PR TITLE
Adds Stuffed Legion

### PR DESCRIPTION
**Pull request by TrustyGun**
:cl:
rscadd: By crafting together a legion skull, a goliath steak, ketchup, and capsaicin oil, you can create Stuffed Legion! Be careful, it's hot.
/:cl:

This PR adds an inactive legion skull that can be dropped by legions, and is used to create Stuffed Legion, along with a goliath steak, ketchup, and capsaicin oil (hotsauce).

legion

It also has some Tricord on the side.

### CODE REVIEW

**Comment by TechnoAlchemisto** [comment]
we really should start adding %drop chance to mining mobs or new drops are going to cause a lot of item bloat for miners

File: code/modules/mob/living/simple_animal/hostile/mining_mobs.dm [hunk 1]
```diff
@@ -733,6 +733,14 @@
 	icon = 'icons/obj/surgery.dmi'
 	icon_state = "demon_heart"
 
+/obj/item/weapon/legion_skull
+	name = "legion's head"
+	desc = "The once living, now empty eyes of the former human's skull cut deep into your soul."
+	icon = 'icons/obj/lavaland/legion_head.dmi'
+	icon_state = "head"
+
+/obj/item/weapon/legion_skull/attack_by
```
Comment (RemieRichards) [inline code/modules/mob/living/simple_animal/hostile/mining_mobs.dm]: this is wrong, you've created a new type called ```/attack_by```

Comment (TrustyGun) [inline code/modules/mob/living/simple_animal/hostile/mining_mobs.dm]: oh whoops, that was an left over from something else

**Comment by PKPenguin321** [comment]
at a glance i thought the sprite was like an animated version of [this guy](https://s3.amazonaws.com/colorslive/png/767309-A43Xf_m2FJrz0_Or.png)

**Comment by PKPenguin321** [comment]
also this is a lot of work for something that hasa *chance* of having like 2u of saline glucose in it
other than that though this is a good PR

**Comment by TechnoAlchemisto** [comment]
You might just want to add tricord instead saline glucose but that's up to you

**Comment by RandomMarine** [comment]
Yeah, considering miners already have abundant access to donk pockets and microwaves, this is really weak in comparison.

**Comment by TechnoAlchemisto** [comment]
considering capsaicin and tomatoes can't be gotten on mining and this requires mining stuff it should get a big buff overall, go crazy it will seldom be made anyways

**Comment by RandomMarine** [comment]
Mining vendor actually had hot chili as a prize.

Long ago.
No idea when someone took it out.

**Comment by Iamgoofball** [comment]
Nah give it omnizine

**Comment by drynwyn** [comment]
Put a Legion Heart in the recipe and make eating it implant it in you.

**Comment by Cheridan** [comment]
I was a little disappointed that this wasn't a cuddly legion plushie.

**Comment by KorPhaeron** [comment]
It's a cute sprite, but I would be surprised if it ever got made considering a full revive drops from the same mob

**Comment by TrustyGun** [comment]
> conflicts

**Comment by KorPhaeron** [comment]
Why is the legion head in its own dmi?

**Comment by TrustyGun** [comment]
it's special and doesn't really fit anywhere else

i could move it I suppose

**Comment by TrustyGun** [comment]
i moved the skull to the mining.dmi file and conflicts has been fixed

**Pull request merged**